src/applications/facility-locator/tests/e2e/facilitySearch.cypress.spec.js

```diff
@@ -171,7 +171,7 @@ describe('Facility VA search', () => {
       });
   });
 
-  it('shows search result header even when no results are found', () => {
+  it.skip('shows search result header even when no results are found', () => {
```

Comment:
Test failed on multiple runs due to a Cypress timeout. This test is also missing an axe check